### DISCUSSION TITLE
开往访问异常

### DISCUSSION THREAD

**Discussion by imqi1-github** [travellings-link/travellings#2444] [寻求帮助] [answered]
点击【随机前往一个开往网页】后跳出Vercel检查点，众所周知Vercel在国内访问速度极慢，可否优化一下呢，非常影响浏览体验，若此问题不改善就退出开往吧，加入也没啥意义了

**Comment by gxres042** [comment]
~~是谁跟你说我们部署站点用的 Vercel 了~~

你好，有 CDN。

![Screenshot_2024-08-06-11-26-00-760_com.android.chrome-edit.jpg](https://github.com/user-attachments/assets/d25db9ba-3b3b-4768-8ffa-7ec9916a0e73)

Vercel 检测点是正常现象。或者，您可以承担一下国内 CDN 被刷流量的相关费用。

**Comment by linlinzzo** [comment]
貌似他们也有分国内国外两个线路，国内现在好像用的是多吉云，访问效果应该会好一点。

前段时间开往一直被刷流量，就我看到开往的赞助资金已经投了七十多块钱到CDN里去。最严重的一天被打了三次欠费。（好像？）所以才不得不改成 Vercel 这种国外服务。

现在貌似又切换回来了？你可以再试试。

> 加入也没啥意义了

这话未免偏激了些

（本人非开往维护组成员，发言仅代表本人观点）

**Comment by Luochancy** [comment, marked as answer]
当时是这样的
被刷流量了

我记得因为刷流量导致cdn欠费直接被掐站，当时垫了七八十块钱进去，毕竟开往是一个非营利性项目。但再怎么说域名的续费跟服务器续费都需要钱，当时的情况并不允许我们再这么把钱嚯嚯到cdn当中，并且还要保持开往的可用性，所以就临时切到Vercel了。

截止今天目前开往应该是不会说再分流到Vercel了，目前也算是停用了这个策略。
